cf-net/cf-net.c

```diff
@@ -704,6 +721,17 @@ static int CFNetGet(ARG_UNUSED CFNetOptions *opts, const char *hostname, char **
                 specified_path = true;
                 break;
             }
+            case 'j':
+            {
+                char *tail = NULL;
+                n_threads = (int) strtol(optarg, &tail, 10);
```

Comment:
I believe @olehermanse has a new function for this purpose.

Reply:
@vpodzime @jimis `StringToLongExitOnError()` from `string_lib.h` would be appropriate. `StringToLong()` if you want to do custom error handling. (I plan to do some changes, like actually returning long on platforms where `long != int`, but they will not be breaking.)